particles/Tutorial/Kotlin/Demo/src/total/TTTRandomComputer.kt

```diff
@@ -22,17 +22,31 @@ import kotlin.native.Retain
 import kotlin.native.internal.ExportForCppRuntime
 
 class TTTRandomComputer : Particle() {
-    private val gameState = Singleton(this, "gameState") { TTTRandomComputer_GameState() }
-    private val myMove = Singleton(this, "myMove") { TTTRandomComputer_MyMove() }
-    private val player = Singleton(this, "player") { TTTRandomComputer_Player() }
+    private val defaultGameState = TTTRandomComputer_GameState(
+        board = ",,,,,,,,",
```

Comment:
Not related to this line, but I wanted to mention that destructuring is now possible given that the values are not null. 

https://kotlinlang.org/docs/reference/data-classes.html#data-classes-and-destructuring-declarations

Reply:
I'm not sure that we want to de-structure the singleton. Could you give an example for why we'd want this?